lexicons/com/atproto/moderation/defs.json

```diff
@@ -34,13 +52,95 @@
       "type": "token",
       "description": "Rude, harassing, explicit, or otherwise unwelcoming behavior"
     },
+    "reasonHarassment": {
+      "type": "token",
+      "description": "Harassment"
+    },
+    "reasonAnimalAbuse": {
+      "type": "token",
+      "description": "Animal Abuse"
+    },
+    "reasonGraphic": {
+      "type": "token",
+      "description": "Graphic: violent or disturbing images"
+    },
+    "reasonScam": {
+      "type": "token",
+      "description": "Scams"
+    },
+    "reasonProhibitedCommerce": {
+      "type": "token",
+      "description": "Prohibited Commerce: trading in banned goods or services"
+    },
+    "reasonInauthentic": {
+      "type": "token",
+      "description": "Inauthentic: bots pretending to be human, fake accounts"
+    },
+    "reasonImpersonation": {
+      "type": "token",
+      "description": "Impersonation: of a specific person or organization"
+    },
+    "reasonBanEvasion": {
+      "type": "token",
+      "description": "Ban Evasion: new accounts from previously taken down users"
+    },
+    "reasonChildSafety": {
+      "type": "token",
+      "description": "Child Safety: CSAM/CSEM, grooming"
+    },
+    "reasonSelfHarm": {
+      "type": "token",
+      "description": "Self-Harm"
+    },
+    "reasonThreat": {
+      "type": "token",
+      "description": "Threat"
+    },
+    "reasonIntolerance": {
+      "type": "token",
+      "description": "Intolerance: slurs, isms"
+    },
+    "reasonPrivateInfo": {
+      "type": "token",
+      "description": "Private Information: doxxing"
+    },
+    "reasonSexualAbuse": {
+      "type": "token",
+      "description": "Sexual Abuse: trafficking, non-consensual intimate images (NCII), deepfake porn"
+    },
+    "reasonSiteSecurity": {
+      "type": "token",
+      "description": "Site Security: account security, hacking"
+    },
+    "reasonTerrorViolentExtremist": {
+      "type": "token",
+      "description": "Terrorist, Violence, and Extremist (TVEC)"
+    },
+    "reasonSpoiler": {
+      "type": "token",
+      "description": "Spoilers (example reason)"
+    },
+    "reasonGrowthHack": {
+      "type": "token",
+      "description": "Growth Hack: engagement farming (example reason)"
+    },
     "reasonOther": {
       "type": "token",
       "description": "Other: reports not falling under another report category"
     },
     "reasonAppeal": {
       "type": "token",
       "description": "Appeal: appeal a previously taken moderation action"
+    },
+    "aspect": {
```

Comment:
It looks like `aspect` and `subjectType` don't use tokens while `reasonType` does use tokens.  My interpretation of the intent here is that the aspect and subject type lists are meant to be specified within `com.atproto` lexicons, while reason types are meant to be extendable by other namespaces.

Seeing values like `chat` and `video` specified within `com.atproto` does give me some pause.  I suppose the idea is that these are generic enough to apply in many settings?

Reply:
Yup, the are closer to an enum or controlled vocabulary, with definitions tied to the reporting system design. While the reason types are extensible via tokens.

In the long run, I think this reporting system should migrate to the `tools.ozone.*` namespace. I was tempted to try and start that with this project, but I think it is too heavy a lift for the time we have. There are some moderation things which do belong in `com.atproto.*` (like labeling, and probably whatever inter-infrastructure takedown delegation and inter-service reporting?), but I think this reporting API is just one take on how reporting should work, and future atproto app developers might invent different systems.